e2e/e2e_suite_test.go

```diff
@@ -43,18 +42,6 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	tmpPath.SetPath(path.Dir(shell.CurrentDir())+"/dist", os.Getenv("PATH"))
 	ensureRequiredBinaries()
 
-	shouldManageCluster := manageCluster()
```

Comment:
In addition `ClientVersion` as-is is not really useful for anything with this change - it will always be `4`. See line `40` here.

https://github.com/Maistra/istio-workspace/blob/1a342b0a7f9dae4c9ceee19037d14c9091a13d25/e2e/infra/cluster_operations.go#L45-L70

Reply:
Implies cleanup in docker registry logic too.

Reply:
90908bf